### PR TITLE
Update Splunk to 7.0.3

### PR DESCRIPTION
**Pull request by Cam-Barker**
Why?
I am submitting this PR because Splunk 7.0.1 is having issues with indexer clusters and Charlie recommended that I upgrade to 7.0.3.

### CODE REVIEW

**Comment by Cam-Barker** [comment]
@nmattam @gravesb Can you review this PR?

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Requirements
 ------------
 * Red Hat Enterprise / CentOS 5.5+ / Windows Server 2008+ (forwarder only) or Ubuntu LTS 12.04+
 * Chef 12+
-
+*
```
Comment (nmattam) [inline README.md]: This was made to kick off the travis build. 6dc58eb18050883af3d211291e2e554d62ed3fae
Let us revert it.

**Comment by Cam-Barker** [comment]
@acharlieh Remove the whole case RUBY_VERSION WHEN '2.1.6' part?

**Comment by acharlieh** [comment]
@Cam-Barker So getting rid of the ruby version block is part of it, telling it how to build / with which dependencies when on ruby 2.1.6, but the .travis.yml file is what drives what Travis builds.

https://github.com/cerner/cerner_splunk/blob/f4a9df3d045efeb1cfeda8e7c5f43345fb9314ef/.travis.yml#L8

File: Gemfile [hunk 2]
```diff
@@ -6,32 +6,6 @@ foodcritic_version = '= 11.0.0'
...
-  # Later versions require Ruby 2.2
-  gem 'nio4r', '< 2.0'
-  # Later versions require Ruby 2.2
-  gem 'rack', '< 2.0'
-  # Later versions (and thus berkshelf) won't load on chef < 12.5
-  # See: https://github.com/berkshelf/ridley/issues/336
-  gem 'ridley', '< 4.4.2'
-else
-  # https://github.com/cerner/cerner_splunk/issues/142
-  chef_version = '= 12.18.31'
```
Comment (acharlieh) [inline Gemfile]: You were a bit overzealous here... "else" = all other cases :)

**Comment by Cam-Barker** [comment]
@nmattam Can you merge this please?

**Pull request merged**